### ISSUE TITLE
Font for right-click menu shouldn't use terminal font

### ISSUE THREAD

**Issue by jacksongoode** [amezin/gnome-shell-extension-ddterm#116]
This is just a suggestion, but I've felt it should be consistent with the tab name font.

**Comment by amezin** [comment]
I agree, but the problem is... I don't know why the menu uses the monospace font. It's not the terminal's font even. I think the issue appeared after some Gtk or Vte update. On Fedora 32-34 VMs, the context menu uses the default UI font

**Comment by amezin** [comment]
VTE test app (`vte-2.91` from the command line) suffers from the same issue. Although GNOME Terminal doesn't.

**Comment by amezin** [comment]
bisected to https://gitlab.gnome.org/GNOME/vte/-/commit/075d0b37049244429b7d709dc8c21f67023bc048

**Comment by amezin** [comment]
https://gitlab.gnome.org/GNOME/vte/-/issues/2542

**Comment by jacksongoode** [comment]
Very cool, very swag :sunglasses:
